src/main/scala/sbtghactions/GenerativeKeys.scala

```diff
@@ -30,6 +30,7 @@ trait GenerativeKeys {
 
   lazy val githubWorkflowSbtCommand = settingKey[String]("The command which invokes sbt (default: sbt)")
   lazy val githubWorkflowUseSbtThinClient = settingKey[Boolean]("Whether to use sbt's native thin client, default is false since this can cause issues (see https://github.com/sbt/sbt/issues/6468)")
+  lazy val githubWorkflowIncludeClean = settingKey[Boolean]("Whether to include the clean.yml file, default is true")
```

Comment:
```suggestion
  lazy val githubWorkflowIncludeClean = settingKey[Boolean]("Whether to include the clean.yml file (default: true)")
```

Just following the pattern of the other settings